test/test_cli.sh

```
@@ -0,0 +1,40 @@
+test_names+=(
+  test_no_args_exits_nonzero
+  test_help_flag_exits_zero
+  test_unknown_command_exits_nonzero
+  test_unknown_option_exits_nonzero
+  test_missing_pr_value_exits_nonzero
+  test_missing_pr_value_stderr_message
+  test_since_flag_not_yet_implemented
+)
+
+test_no_args_exits_nonzero() {
+  assert_exit 1 "no args should exit 1" bash "$script"
```

Shellcheck (0.11.0)

[warning] script is referenced but not assigned.

(SC2154)